plugins/telink-dfu/fu-telink-dfu-archive.c

```diff
@@ -17,6 +17,11 @@ G_DEFINE_TYPE(FuTelinkDfuArchive, fu_telink_dfu_archive, FU_TYPE_FIRMWARE)
 
 #define FU_TELINK_DFU_FIRMWARE_JSON_FORMAT_VERSION_MAX 0
 
+typedef struct bl_type_s {
+	const gchar *name;
+	FwupdVersionFormat ver_format;
+} bl_type_t;
```

Comment:
`FuTelinkDfuBlType` please

Reply:
resolved in commit e62e6dc36415bfa48d780538a4c94b4ac2db5c72